package.json

```diff
@@ -45,6 +45,7 @@
     "@types/subleveldown": "^4.1.0",
     "chai": "^4.3.4",
     "chai-as-promised": "^7.1.1",
+    "del": "^6.0.0",
```

Comment:
had to install this module because `rmdirAsync` from fs module wasn't compatible with node v10 for recursively deleting a folder.

Reply:
Can we remove this and use `rmdirAsync` once https://github.com/thehubbleproject/hubble-contracts/issues/567 is completed? If so I would add a comment to that issue to that effect.

Reply:
Yeah sure we can